### ISSUE TITLE
Resetting an already resetted sim takes the soluteType to liquid then back to solid

### ISSUE THREAD

**Issue by samreid** [phetsims/concentration#39]
Resetting an already resetted sim takes the soluteType to liquid then back to solid.

```js
{
  "messageIndex": 139,
  "eventType": "user",
  "componentID": "concentrationScreen.resetAllButton",
  "componentType": "ResetAllButton",
  "event": "fired",
  "time": 1428079706952,
  "children": [
    {
      "messageIndex": 140,
      "eventType": "model",
      "componentID": "concentrationScreen.soluteForm",
      "componentType": "Property",
      "event": "changed",
      "time": 1428079706952,
      "options": {
        "oldValue": "solid",
        "newValue": "liquid"
      }
    },
    {
      "messageIndex": 141,
      "eventType": "model",
      "componentID": "concentrationScreen.soluteForm",
      "componentType": "Property",
      "event": "changed",
      "time": 1428079706952,
      "options": {
        "oldValue": "liquid",
        "newValue": "solid"
      }
    }
  ]
}
```

This should be resolved for together integration within the next 2 weeks or so.

**Comment by pixelzoom** [comment]
This was broken by https://github.com/phetsims/beers-law-lab/issues/89.  Fixed in 6912d01264df3f6d149841cd58b72f38e8ad0fac.  @samreid please verify and close.

**Comment by samreid** [comment]
I tested the arch output and it is fixed, and the change set looks good.  Closing.
